File: CS/dx_sample/Form1.cs
using DevExpress.XtraReports;
using DevExpress.XtraReports.UI;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace dx_sample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            XtraReport1 x = new XtraReport1();
            x.DataSource = TestData.CreateRows();
            ReportPrintTool pt = new ReportPrintTool(x);
            pt.AutoShowParametersPanel = true;
            pt.ShowPreviewDialog();
        }
    }
}

File: VB/dx_sample/Form1.vb
Imports DevExpress.XtraReports
Imports DevExpress.XtraReports.UI
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace dx_sample
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			Dim x As New XtraReport1()
			x.DataSource = TestData.CreateRows()
			Dim pt As New ReportPrintTool(x)
			pt.AutoShowParametersPanel = True
			pt.ShowPreviewDialog()
		End Sub
	End Class
End Namespace
